### PR TITLE
andaya module

### PR DESCRIPTION
**Pull request by andaya-gener**
Add andaya module and call it to the main.py

It contains:
1. My Basic Information. 
2. Goals.
3. Comments from teammates.

### CODE REVIEW

File: ourpackage/andaya.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+def gener():
+    while True:
+        os.system('cls')
+        print("================= Andaya's Menu ==================")
+        print("=                                                =")
+        print("= Hello Everyone! My name is Gener A. Andaya Jr. =")
+        print("=                                                =")
+        print("= 1. My basic info.                              =")
+        print("= 2. My personal goals.                          =")
+        print("= 3. Comment from teammate 1.                    =")
```
Comment (barcelos-kevin) [inline ourpackage/andaya.py]: Add my own comment. - Barcelos

**Review by barcelos-kevin — CHANGES_REQUESTED** [review]

File: ourpackage/andaya.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        os.system('cls')
+        print("================= Andaya's Menu ==================")
+        print("=                                                =")
+        print("= Hello Everyone! My name is Gener A. Andaya Jr. =")
+        print("=                                                =")
+        print("= 1. My basic info.                              =")
+        print("= 2. My personal goals.                          =")
+        print("= 3. Comment from Barcelos.                      =")
+        print("= 4. Comment from Veslino.                       =")
+        print("= 5. Comment from teammate 3.                    =")
```
Comment (consultado-kirby) [inline ourpackage/andaya.py]: I will add my comment. - Consultado

**Review by consultado-kirby — CHANGES_REQUESTED** [review]

File: ourpackage/andaya.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        print("================= Andaya's Menu ==================")
+        print("=                                                =")
+        print("= Hello Everyone! My name is Gener A. Andaya Jr. =")
+        print("=                                                =")
+        print("= 1. My basic info.                              =")
+        print("= 2. My personal goals.                          =")
+        print("= 3. Comment from Barcelos.                      =")
+        print("= 4. Comment from Veslino.                       =")
+        print("= 5. Comment from Consultado.                    =")
+        print("= 6. Comment from teammate 4.                    =")
```
Comment (piadozo-edriane) [inline ourpackage/andaya.py]: I would like to add a comment into your code - Piadozo

**Review by piadozo-edriane — CHANGES_REQUESTED** [review]

File: ourpackage/andaya.py [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    while True:
+        os.system('cls')
+        print("================= Andaya's Menu ==================")
+        print("=                                                =")
+        print("= Hello Everyone! My name is Gener A. Andaya Jr. =")
+        print("=                                                =")
+        print("= 1. My basic info.                              =")
+        print("= 2. My personal goals.                          =")
+        print("= 3. Comment from Barcelos.                      =")
+        print("= 4. Comment from Veslino.                       =")
```
Comment (veslino-marc) [inline ourpackage/andaya.py:13]: I will add my comment. - Veslino

**Review by veslino-marc — CHANGES_REQUESTED** [review]

File: ourpackage/andaya.py (hunk 1 above)
Comment (andaya-gener) [reply ourpackage/andaya.py]: You have done it.

File: ourpackage/andaya.py (hunk 2 above)
Comment (andaya-gener) [reply ourpackage/andaya.py]: You have done it.

File: ourpackage/andaya.py (hunk 3 above)
Comment (andaya-gener) [reply ourpackage/andaya.py]: You have done it.

File: ourpackage/andaya.py (hunk 4 above)
Comment (andaya-gener) [reply ourpackage/andaya.py:13]: You have done it.

**Review by consultado-kirby — APPROVED** [review]

**Review by barcelos-kevin — APPROVED** [review]
Looks good to me.

**Review by veslino-marc — APPROVED** [review]

**Review by piadozo-edriane — APPROVED** [review]
Thank you :)

**Pull request merged**